frontend/src/component/common/FormTemplate/FormTemplate.tsx

```diff
@@ -28,6 +28,130 @@ interface ICreateProps {
     formatApiCode: () => string;
 }
 
+const StyledContainer = styled('section', {
+    shouldForwardProp: prop => prop !== 'modal',
+})<{ modal?: boolean }>(({ theme, modal }) => ({
+    minHeight: modal ? '100vh' : '80vh',
+    borderRadius: modal ? 0 : theme.spacing(2),
+    width: '100%',
+    display: 'flex',
+    margin: '0 auto',
+    overflow: 'hidden',
+    [theme.breakpoints.down(1100)]: {
+        flexDirection: 'column',
+        minHeight: 0,
+    },
+}));
+
+const StyledRelativeDiv = styled('div')(({ theme }) => ({
+    ...relative,
+}));
+
+const StyledFormContent = styled('div')(({ theme }) => ({
+    backgroundColor: theme.palette.formBackground,
+    display: 'flex',
+    flexDirection: 'column',
+    padding: theme.spacing(6),
+    flexGrow: 1,
+    [theme.breakpoints.down(1200)]: {
```

Comment:
could we use theme.breakpoints.sm|lg etc or would it look weird? They are very similar values but not exactly the same